convert2rhel/unit_tests/utils_test.py

```diff
@@ -37,6 +37,7 @@
 from convert2rhel import unit_tests  # Imports unit_tests/__init__.py
 from convert2rhel import utils
 from convert2rhel.systeminfo import system_info
+from convert2rhel.toolopts import ToolOpts
```

Comment:
This import doesn't seem to be used anywhere in the `utils_test.py` file. Is it needed by anything else? 
```suggestion

```